dependencies_ros1.rosinstall

```diff
@@ -1,3 +1,4 @@
 # PCL ros tools, this is required because using custom version of PCL and VTK
 - git: {local-name: perception_pcl, uri: 'https://github.com/ros-perception/perception_pcl.git', version: 1.7.1}
 - git: {local-name: cmake_common_scripts, uri: 'https://github.com/ros-industrial/cmake_common_scripts.git', version: master}
+- git: {local-name: heat_raster, uri: 'https://github.com/ros-industrial/heat_raster.git', version: master}
```

Comment:
Should this be `https://github.com/swri-robotics/heat_raster.git`?

Reply:
Yes, I updated